src/Common/WellKnown/KnownSymbols.cs

```diff
@@ -22,21 +22,11 @@ public KnownSymbols(Compilation compilation)
     {
     }
 
-    /// <summary>
-    /// Gets the class <c>System.Threading.Tasks.Task</c>.
-    /// </summary>
-    public INamedTypeSymbol? Task => TypeProvider.GetOrCreateTypeByMetadataName("System.Threading.Tasks.Task");
```

Comment:
Are you sure you really want to remove this (and the ValueTask version)? I assume this is because they're currently unused, but it feels like you'll just be adding them back soon.

Reply:
I added tests for ValueTask<T>. When I write code that takes Task or ValueTask (without the return type) they always run into the path to exit out of the analyzer using the generic version. If we need them again, we can add them (and we'll have coverage)